content/en/docs/collector/configuration.md

```diff
@@ -8,14 +8,40 @@ cSpell:ignore: cfssl cfssljson fluentforward gencert genkey hostmetrics initca l
 
 <!-- markdownlint-disable link-fragments -->
 
-You can configure the Collector to suit your observability needs. Before you
-learn how Collector configuration works, familiarize yourself with the following
-content:
+You can configure the OpenTelemetry Collector to suit your observability needs.
+Before you learn how Collector configuration works, familiarize yourself with
+the following content:
 
-- [Data collection concepts][dcc] in order to understand the repositories
-  applicable to the OpenTelemetry Collector.
+- [Data collection concepts][dcc], to understand the repositories applicable to
+  the OpenTelemetry Collector.
 - [Security guidance](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md)
 
+## Location {#location}
+
+By default, the Collector configuration is located in
+`/etc/otelcol/config.yaml`.
```

Comment:
I think this is only true for `core`, if I remember correctly for contrib it was `/etc/otelcol-contrib/config.yaml`?

Reply:
We'll definitely want to list the contrib location if it's different, since most people use contrib

Reply:
> We'll definitely want to list the contrib location if it's different, since most people use contrib

... although they shouldn't ...

Reply:
I disagree, they absolutely should. There's just too many useful things in there that you'll need, especially when there's several sources of data and a need to normalize or transform a bit.

Reply:
Would it help to write something like...

```
By default, the Collector configuration is located in `/etc/<otel-directory>/config.yaml`, where `<otel-directory>` can be `otelcol`, `otelcol-contrib`, or another value, depending on the Collector version or distribution you're using.
```

Reply:
> I disagree, they absolutely should. There's just too many useful things in there that you'll need, especially when there's several sources of data and a need to normalize or transform a bit.

The recommendation we should eventually have for that is that people use `ocb` to have the collector tailored to exactly the components they need. This is mainly a security concern. So it's more about "Use otelcol-contrib to figure out what you need, then use `ocb` to create the collector that only contains that", but that's something we should put into a dedicated issue eventually